.github/workflows/plugin-update.yml

```diff
@@ -39,20 +39,27 @@ jobs:
           cache: false # save cache space for vault builds: https://github.com/hashicorp/vault/pull/21764
           go-version-file: .go-version
 
-      - name: update plugin
+      - name: Update plugin
+        if: github.repository == 'hashicorp/vault'
         run: |
           go get "github.com/hashicorp/${{ inputs.plugin }}@v${{ inputs.version }}"
           go mod tidy
 
-      - name: detect changes
+      - name: Update Enterprise-only plugin
+        if: github.repository == 'hashicorp/vault-enterprise'
+        run: |
+          (cd vault_ent && go get "github.com/hashicorp/${{ inputs.plugin }}@v${{ inputs.version }}" && go mod tidy)
```

Comment:
Doubling up `go mod tidy` here?

Reply:
That's intentional. We run tidy in both the `vault_ent` dir and the root vault directory. This line is slightly obtuse because of the parenthesis which run everything in a subshell. I am happy to change it to be more explicit if that is preferred.

Reply:
you could use `pushd` then `popd` to avoid the subshell

Reply:
I'm fine as long as it's intentional.